.github/workflows/release.yml

```diff
@@ -0,0 +1,79 @@
+# build, test and release push-to-cloud
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: branch, tag or SHA to checkout.
+        required: false
+        default: 'master'
+      version:
+        description: semantic version to release.
+        required: true
+      release_note:
+        description: release note file.
```

Comment:
Are we thinking that we'll commit/store changelog files in the repo? Or would they just be throwaway files so we can format the release notes easier than if it was a plain argument?

Reply:
I'd have thought we'd commit them to the repo under docs/release/VERSION

Reply:
Ah I gotcha, I blanked out and read the path you gave in the PR description as just ideas for different directories instead. If we wanted to require notes for a release, we could use the version argument to infer the changelog location. Maybe that would be over-engineering this, I dunno.